ember_debug/vendor/startup-wrapper.js

```diff
@@ -23,6 +23,11 @@ if (typeof env !== 'undefined') {
 var EMBER_VERSIONS_SUPPORTED = {{EMBER_VERSIONS_SUPPORTED}};
 // @formatter:on
 
+// Enable debug render tree
+
+window.EmberENV = window.EmberENV || {};
+window.EmberENV._DEBUG_RENDER_TREE = true;
```

Comment:
Do you know what is the timing of this script's injection?

Reply:
I do not

Reply:
Hm we will have to figure that out. It will not work correctly if it's set after Ember is loaded.

Reply:
We have a function under this that loads on Ember load, so I am guessing if we define stuff before it, it should get set before Ember is done, but not necessarily before Ember starts. Is there a way to log when Ember starts loading?